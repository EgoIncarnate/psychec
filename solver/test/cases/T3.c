int main() {
    T t;
    t.i = 0;
    t.next = &t;

    T2 t2;
    t2.i = 10;
    t2.next = &t2;
}
