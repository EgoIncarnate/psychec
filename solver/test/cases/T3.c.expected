typedef struct __T2__ {int i; struct __T2__ * next;} T2 ;
typedef struct __T__ {struct __T__ * next;uintptr_t i; } T ;
